### PR TITLE
add newrelic

### PR DESCRIPTION
**Pull request by raskchanky**
ping @chef/chef-visual-interaction

### CODE REVIEW

**Comment by raskchanky** [comment]
This failure looks transient.  Everything passes locally for me.  I'll start a new build.

File: src/oc-id/.gitignore [hunk 1]
```diff
@@ -22,6 +22,7 @@ vendor/ruby
 config/settings.local.yml
 config/settings/*.local.yml
 config/environments/*.local.yml# RailsConfig ignores
+config/newrelic.yml
```
Comment (smith) [inline src/oc-id/.gitignore]: Should we do the same thing I [mentioned on account management](https://github.com/chef/account_management/pull/107#issuecomment-104389791)?

**Comment by smith** [comment]
:+1:

File: src/oc-id/.gitignore (hunk 1 above)
Comment (raskchanky) [inline src/oc-id/.gitignore]: Yes, will do.

**Pull request merged**